catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrCatalogProvider.java

```diff
@@ -286,7 +288,7 @@ public CreateResponse create(CreateRequest request) throws IngestException {
         }
 
         pendingNrtIndex.putAll(output.stream()
-                .collect(Collectors.toMap(Metacard::getId, Function.identity())));
+                .collect(Collectors.toMap(Metacard::getId, m -> copyMetacard(m))));
```

Comment:
Is defensive logic something that we should unit test? Seems like the kind of thing that would be easy to break by accident.